collectors/python.d.plugin/isc_dhcpd/README.md

```diff
@@ -12,7 +12,7 @@ Monitors the leases database to show all active leases for given pools.
 
 -   dhcpd leases file MUST BE readable by Netdata
 -   pools MUST BE in CIDR format
--   'python-ipaddress' package is needed
+-   'python-ipaddress' package is needed in Python2
```

Comment:
use ``

'python-ipaddress' => `python-ipaddress`

Reply:
what do you mean by this?

Reply:
i mean use `code block` format. (`` instead of '')